internal/backends/winit/glwindow.rs

```diff
@@ -725,17 +731,15 @@ impl<Renderer: WinitCompatibleRenderer + 'static> WindowAdapterSealed for GLWind
     }
 
     fn set_size(&self, size: corelib::api::WindowSize) {
+        if self.in_resize_event.get() {
+            return;
+        }
         if let Ok(mut map_state) = self.map_state.try_borrow_mut() {
```

Comment:
so we don't need the _mut here anymore?  (that's what i read from the comment bellow)

Reply:
The mut is still need for the assignment in line 740, I think. The comment was outdated.